terraform/executeTerraformCleanup.sh

```diff
@@ -48,6 +48,8 @@ case "$service" in
     ;;
     "EKS_ADOT_OPERATOR") TEST_FOLDER="./eks/";
     ;;
+    "EKS_ADOT_OPERATOR_ARM64") TEST_FOLDER="./eks/";
```

Comment:
```suggestion
    "EKS_ADOT_OPERATOR_ARM64") TEST_FOLDER="./eks/";
```
Don't we also need to set the cluster name here?

Reply:
Unsure - if yes, does cluster name also need to be set in for `EKS_ARM64` test cleanup?